test/files/presentation/ypredef-completion-implicit-chained/src/Completions.scala

```diff
@@ -0,0 +1,12 @@
+package test
+
+import scala.Predef.DummyImplicit // turn off other predef implicits for a cleaner .check file.
+
+object O {
+  def map(x: Int => Int)(implicit a: DummyImplicit): O.type = this
+  val prefix123 : Int = 0
+}
+
+class Foo {
+  O.map(x => x)./*!*/ // we want the presentation compiler to apply the implicit argument list.
```

Comment:
This test was a blocker for the initial iterations/implementation of `-Ypredef`, so I wanted to ensure that it passed when I finalized the approach.